code/modules/surgery/bodyparts/bodyparts.dm

```diff
@@ -197,8 +197,8 @@
 //Damage cannot go below zero.
 //Cannot remove negative damage (i.e. apply damage)
 /obj/item/bodypart/proc/heal_damage(brute, burn, stamina, required_status, updating_health = TRUE)
-
-	if(required_status && (status != required_status)) //So we can only heal certain kinds of limbs, ie robotic vs organic.
+	// yogs -- line below updated to allow for robotic body part healing override
```

Comment:
could just put it at the end of the other comment

Reply:
comment is long enough as is